au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/BalanceKnob.qml

```diff
@@ -0,0 +1,62 @@
+/*
+* Audacity: A Digital Audio Editor
+*/
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+import Muse.Ui 1.0
+import Muse.UiComponents 1.0
+
+KnobControl {
+    id: root
+    isBalanceKnob: true
+
+    StyledPopupView {
```

Comment:
It looks like this tooltip would be better made as a separate component, it will probably be needed in other places

Reply:
Done